CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.6.2
+ - Fix bug where update index actions would not work with events with 'data' field
```

Comment:
This patch doesn't affect user behavior. It fixes the bug where events with a data field wouldn't work.

Am I missing something?